scapy/layers/dot11.py

```diff
@@ -378,7 +378,7 @@ class RadioTap(Packet):
             FlagsField("PresentVHT", None, -8, _rt_presentvht),
             lambda pkt: pkt.present and pkt.present.VHT),
         ConditionalField(
-            ByteEnumField("bandwidth", 0, _rt_vhtbandwidth),
+            ByteEnumField("VHT_bandwidth", 0, _rt_vhtbandwidth),
```

Comment:
Leave VHT’s bandwidth called `bandwidth` for backward compatibility
```suggestion
            ByteEnumField("bandwidth", 0, _rt_vhtbandwidth),
```